core/blaze/blaze_driver.F90

```diff
@@ -177,14 +174,15 @@ SUBROUTINE BLAZE_DRIVER ( NCELLS, BLAZE, SF, casapool,  casaflux, casamet, &
   DO i = 1, BLAZE%NCELLS
      DO p = 1, landpt(i)%nap  ! loop over number of active patches
         patch_index = landpt(i)%cstart + p - 1 ! patch index in CABLE vector
-        WRITE(900+BLAZE%IAM,*)" Pidx",patch_index,patch(:)%frac
-
+ 
         IF ( casamet%lnonwood(patch_index) == 1 ) THEN ! Here non-wood
-
-           casaflux%kplant_fire(patch_index,LEAF)  = real(BLAZE%AB(i), r_2) &
-                * real(casapool%cplant(patch_index,LEAF )*patch(patch_index)%frac, r_2)
-           casaflux%kplant_fire(patch_index,FROOT) = real(BLAZE%AB(i), r_2) &
-                * real(casapool%cplant(patch_index,FROOT)*patch(patch_index)%frac, r_2)
+!CLN wrong for kplant. Take out cmass here
```

Comment:
I think this is undoing a BLAZE related change - we need to check

Reply:
And can you advise on this one too please?

Reply:
I think we need to look at the revision history for this one - my suspicion is that this was implemented in the BLAZE_2pt9_BLAZE branch (by Lars) after BLAZE_9184 branched off and so should be retained in the merge.

i.e. this is correct
```
casaflux%kplant_fire(patch_index,LEAF)  = real(BLAZE%AB(i), r_2)
casaflux%kplant_fire(patch_index,FROOT) = real(BLAZE%AB(i), r_2) 
```

Reply:
Okay.  The correct code is in the Spin_1951 branch, but not in BLAZE_9184 so it will be updated in BLAZE_9184 after the merge.